base/utils/DESCRIPTION

```diff
@@ -23,38 +23,36 @@ Description: The Predictive Ecosystem Carbon Analyzer
     investigation.
 Imports:
     abind (>= 1.4.5),
-    coda (>= 0.18),
-    data.table,
     dplyr,
-    ggplot2,
     lubridate (>= 1.6.0),
     magrittr,
     ncdf4 (>= 1.15),
     PEcAn.logger,
     PEcAn.remote,
-    PeriodicTable,
-    plyr (>= 1.8.4),
     purrr,
-    randtoolbox,
-    raster,
     RCurl,
-    rjags,
-    sp,
     stringi,
     udunits2 (>= 0.11),
-    xtable,
     XML
 Suggests:
+    coda (>= 0.18),
     MASS,
+    data.table,
+    ggplot2,
     PEcAn.data.atmosphere,
     PEcAn.data.land,
     PEcAn.emulator,
     PEcAn.settings,
     PEcAn.DB,
-    testthat (>= 2.0.0)
+    randtoolbox,
+    raster,
+    rjags,
+    sp,
+    testthat (>= 2.0.0),
+    xtable
 License: FreeBSD + file LICENSE
 Copyright: Authors
 LazyData: true
-Require: hdf5, plyr
+Require: hdf5
```

Comment:
I think R ignores this field. Should this be SystemRequirements instead?

Reply:
I'm just gonna drop it altogether. All of the system requirements are for dependent packages, so it's probably their responsibility to make sure this is documented.